<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWidget</class>
 <widget class="QWidget" name="settingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;
border-color: rgb(196, 178, 255);
border-color: rgb(170, 0, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>设置</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>邮箱</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>271</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 1px  solid;
     border-radius: 4px;
	font: 24px
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>271</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 1px  solid;
     border-radius: 4px;
	font: 24px
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>79</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>271</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 1px  solid;
     border-radius: 4px;
	font: 24px
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="avatarLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string>(头像)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pickAvatarBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #3366FF;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#4f85e2;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
    </property>
    <property name="text">
     <string>更换头像</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showPwdCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>显示密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>重复新密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>259</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>新密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatNewPwdEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>308</y>
      <width>271</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 1px  solid;
     border-radius: 4px;
	font: 24px
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPwdEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>258</y>
      <width>271</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 1px  solid;
     border-radius: 4px;
	font: 24px
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #3366FF;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#4f85e2;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>128</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #3366FF;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#4f85e2;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
